docs/asciidoc/common-problems.adoc

```diff
@@ -259,7 +259,7 @@ For e.g. One could move Swagger UI under `/documentation` using this code.
 	@Override
 	public void addViewControllers(ViewControllerRegistry registry) {
 	
-		registry.addRedirectViewController("/documentation/v2/api-docs", "/v2/api-docs?group=restful-api");
+		registry.addRedirectViewController("/documentation/v2/api-docs", "/v2/api-docs").setKeepQueryParams(true);
```

Comment:
Apply it to all the redirects?

```suggestion
		registry.addRedirectViewController(
				"/documentation/v2/api-docs", 
				"/v2/api-docs")
			.setKeepQueryParams(true);
        	registry.addRedirectViewController(
				"/documentation/swagger-resources/configuration/ui",
				"/swagger-resources/configuration/ui")
			.setKeepQueryParams(true);
        	registry.addRedirectViewController(
				"/documentation/swagger-resources/configuration/security",
				"/swagger-resources/configuration/security")
			.setKeepQueryParams(true);
        	registry.addRedirectViewController(
				"/documentation/swagger-resources", 
				"/swagger-resources")
			.setKeepQueryParams(true);
```

Reply:
Sure, but I think that today only the first redirect uses query parameters.

Reply:
Oh thats right... thanks for pointing that out.